README.md

```diff
@@ -1,229 +1,116 @@
-<div>
-  <a href="https://github.com/CopilotKit/CopilotKit/tree/main/examples/coagents-travel"><code>GitHub →</code></a>
-  <a href="https://docs.copilotkit.ai/coagents/tutorials/ai-travel-app/overview"><code>Tutorial →</code></a>
-</div>
+## Getting Started
 
-### 🔍 [Research Canvas](https://github.com/CopilotKit/CopilotKit/tree/main/examples/coagents-research-canvas)
-Multi-agent document analysis system that helps users analyze papers, synthesize information, and generate comprehensive research summaries through collaborative AI workflows.
-<div>
-  <a href="https://github.com/CopilotKit/CopilotKit/tree/main/examples/coagents-research-canvas"><code>GitHub →</code></a>
-  <a href="https://examples-coagents-research-canvas-ui.vercel.app
-"><code>Live Demo →</code></a>
-</div>
+Begin in minutes with our [Quickstart Guide](https://docs.copilotkit.ai/quickstart).
 
-# Getting Started
-Get started in minutes - check out the [quickstart documentation](https://docs.copilotkit.ai/quickstart).
+## Code Samples
 
-# Code Samples
 ```ts
 // Headless UI with full control
-const { visibleMessages, appendMessage, setMessages, ... } = useCopilotChat();
+const { visibleMessages, appendMessage, setMessages } = useCopilotChat();
 
-// Pre-built components with deep customization options (CSS + pass custom sub-components)
-<CopilotPopup 
-  instructions={"You are assisting the user as best as you can. Answer in the best way possible given the data you have."} 
-  labels={{ title: "Popup Assistant", initial: "Need any help?" }} 
+// Pre‑built component with deep customization
+<CopilotPopup
+  instructions="You are assisting the user."
+  labels={{ title: "Assistant", initial: "Need help?" }}
 />
 
-// ---
-
-// Frontend RAG
+// Frontend Retrieval‑Augmented Generation
 useCopilotReadable({
   description: "The current user's colleagues",
   value: colleagues,
 });
 
-// knowledge-base integration
-useCopilotKnowledgebase(myCustomKnowledgeBase)
-
-// ---
-
-// Frontend actions + generative UI, with full streaming support
-useCopilotAction({
-  name: "appendToSpreadsheet",
-  description: "Append rows to the current spreadsheet",
-  parameters: [
-    { name: "rows", type: "object[]", attributes: [{ name: "cells", type: "object[]", attributes: [{ name: "value", type: "string" }] }] }
-  ],
-  render: ({ status, args }) => <Spreadsheet data={canonicalSpreadsheetData(args.rows)} />,
-  handler: ({ rows }) => setSpreadsheet({ ...spreadsheet, rows: [...spreadsheet.rows, ...canonicalSpreadsheetData(rows)] }),
-});
-
-// ---
-
-// structured autocomplete for anything
-const { suggestions } = useCopilotStructuredAutocompletion(
-  {
-    instructions: `Autocomplete or modify spreadsheet rows based on the inferred user intent.`,
-    value: { rows: spreadsheet.rows.map((row) => ({ cells: row })) },
-    enabled: !!activeCell && !spreadsheetIsEmpty,
-  },
-  [activeCell, spreadsheet]
-);
+// Knowledge‑base integration
+useCopilotKnowledgebase(myCustomKnowledgeBase);
```

Comment:
@suhasdeshpande this doesn't exist thought, right? Outside of this I think in general we can come up with a better code snippet. What do you think?

If we want to add the trendshift badge it may be worth *only* doing that in this PR.